spark-cassandra-connector/src/it/scala/com/datastax/spark/connector/sql/CassandraDataFrameSpec.scala

```diff
@@ -200,61 +192,4 @@ class CassandraDataFrameSpec extends SparkCassandraITFlatSpecBase {
       session.execute(s"select count(1) from $ks.tuple_test2").one().getLong(0) should be (1)
     }
   }
-
-  it should "read and write C* LocalDate columns" in {
```

Comment:
The original version was working, but it was changed to this:
https://github.com/datastax/spark-cassandra-connector/pull/993/commits/e66fccc7be053309e25750f13047fdc63740ff37
Which was not passing on Jenkins - builds:1025, 1026